src/components/CardComponent.js

```diff
@@ -6,34 +6,61 @@ import CardContent from '@material-ui/core/CardContent'
+      <Card className={classes.card}>
+        <CardMedia
+          className={classes.cardMedia}
+          image='data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22288%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20288%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_164edaf95ee%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A14pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_164edaf95ee%22%3E%3Crect%20width%3D%22288%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2296.32500076293945%22%20y%3D%22118.8%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E' // eslint-disable-line max-len
+          title='Image title'
+        />
+        <CardContent className={classes.cardContent}>
+          <Typography gutterBottom variant='h5' component='h2'>
+            Heading
 
-export default ({ card, classes }) => {
-  return (<Grid item key={card} sm={6} md={4} lg={3}>
-    <Card className={classes.card}>
-      <CardMedia
-        className={classes.cardMedia}
-        image='data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22288%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20288%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_164edaf95ee%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A14pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_164edaf95ee%22%3E%3Crect%20width%3D%22288%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2296.32500076293945%22%20y%3D%22118.8%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E' // eslint-disable-line max-len
-        title='Image title'
-      />
-      <CardContent className={classes.cardContent}>
-        <Typography gutterBottom variant='h5' component='h2'>
-          Heading
         </Typography>
-        <Typography>
-          This is a media card. You can use this section to describe the content.
+          <Typography>
+            This is a media card. You can use this section to describe the content.
         </Typography>
-      </CardContent>
-      <CardActions>
-        <Button size='small' color='primary'>
-          View
+        </CardContent>
+        <CardActions>
+          <Button size='small' color='primary' onClick={() => this.openJobModal()}>
+            View
         </Button>
-        <Button size='small' color='primary'>
-          Edit
+          {/* <JobModal isOpen={this.state.isModalOpen} onClose={() => this.closeJobModal()}></JobModal> */}
+          <JobModalNew isOpen={this.state.isModalOpen} onClose={() => this.closeJobModal()}></JobModalNew>

```

Comment:
you're `calling` this function `onClose` and then defining `onClose` as a separate function inside `JobModalNew` - that doesn't really make sense.  You should pass the new `toggleModal` method as `onClose` prop -- pass, not call.  Then in the JobModalNew component you can call the method on close.

Reply:
Just to inform, 'onClose' is a prop of Modal api that expects a callback function that is fired when the component is closed.
https://material-ui.com/api/modal/